CONTRIBUTING.md

```diff
@@ -0,0 +1,137 @@
+# Contributing to Azure Functions
+
+Azure Functions welcomes contributions and suggestions. One of the easiest ways to contribute is to participate in discussions on GitHub issues. You can also contribute by submitting pull requests with code changes or documentation updates. If you have any questions, please ask a project maintainer.
+
+Not all repositories related to Azure Functions adhere to the processes discussed in this guide. In interacting with any repository, it is suggested that you confirm the specific contribution guidelines and process for that repo. This guide covers the baseline standards for most repositories managed by the Azure Functions team.
+
+Contents:
+- [Code of conduct](#code-of-conduct)
+- [Reporting issues](#reporting-issues)
+- [Submitting a pull request](#submitting-a-pull-request)
+- [Creating documentation and samples](#creating-documentation-and-samples)
+- [Repository index](#azure-functions-repositories)
+
+## Code of conduct
+
+Azure Functions projects have adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/).  For more information, see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
+
+## Reporting issues
+
+For non-security related bugs, please log an issue on the appropriate GitHub repository. If you're not sure which repository to go to or have a general issue, please log an issue on the [Azure/Azure-Functions issue tracker](https://github.com/Azure/Azure-Functions/issues).
+
+### Reporting security issues and bugs
+Security issues and bugs should be reported privately, via email, to the Microsoft Security Response Center (MSRC): secure@microsoft.com. You should receive a response within 24 hours. If for some reason you do not, please follow up via email to ensure we received your original message. Further information, including the MSRC PGP key, can be found in the [Security TechCenter](https://technet.microsoft.com/en-us/security/ff852094.aspx).
+
+### Label glossary
```

Comment:
I suggest moving all glossary sections to the end of this guide. These aren't generally actionable by our customers. Instructions for pull requests, etc. should be higher up.

Reply:
Good call. Will get that in the next iteration.